src/universal/components/NewMeetingSidebarPhaseListItem.js

```diff
@@ -1,11 +1,12 @@
+import type {Node} from 'react';
 // @flow
 import React from 'react';
 import styled, {css} from 'react-emotion';
 import appTheme from 'universal/styles/theme/appTheme';
 import ui from 'universal/styles/ui';
+import {phaseLabelLookup} from 'universal/utils/meetings/lookups';
 
 const NavListItem = styled('li')({
-  fontSize: 0,
```

Comment:
hey @ackernaut what's the purpose of fontSize: 0? i had to remove it here, but i saw it in other places. just curious.

Reply:
It was an artifact from using `inline-block` to lay inner elements out LTR. We should be using flex if we aren’t already to handle this.